denorm/denorms.py

```diff
@@ -74,7 +74,7 @@ def get_alldenorms():
     alldenorms = []
     for model in gmodels.get_models(include_auto_created=True):
         if not model._meta.proxy:
-            for field in model._meta.fields + model._meta.local_many_to_many:
+            for field in tuple(model._meta.fields) + tuple(model._meta.local_many_to_many):
```

Comment:
1. You usually control this via the `Meta` class created into the models to change its behaviours. It becomes `_meta` later. So, the `Meta.fields` you put there changes to `_meta.initial_fields`, if I remember correctly.
2. The thing is a tuple by default. However, is usual to set the `Meta.fields` as a list and not a tuple, to ease dynamic customisation on the `__init__` of forms and admins. But then it does not sum anymore if left and right are of different types (list and tuple).